xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -356,7 +356,7 @@
             Useful for handling values that do not fit in the <see cref="T:System.Int64" /> type.
             </remarks>
         <exception cref="T:System.InvalidOperationException">The next data item does not have the correct major type.</exception>
-        <exception cref="T:System.OverflowException">The encoded integer is out of range for <see cref="T:System.UInt32" /></exception>
+        <exception cref="T:System.OverflowException">The encoded integer is out of range for <see cref="T:System.UInt64" /></exception>
```

Comment:
Actually I think this is erroneous, the method will either throw `InvalidOperationException` if it's not a negative number or a `CborContentException` because the CBOR encoding is invalid.
```suggestion
```